utils/dynamodb/formatResponse.js

```diff
@@ -0,0 +1,140 @@
+import { DYNAMO_DB_ATTRIBUTE_TYPES, DYNAMO_DB_ID_SEPARATOR, DYNAMO_DB_TYPES } from "@/constants";
+
+// Formats image to its correct form 
+// returns an object { key, name, link }
+export const formatImage = (image) => {
+  const imageUrl = process.env.KS_S3_BASE_URL;
+  let originalName = '';
+  if (image.includes("_")) {
+    originalName = image.split("_")[1];
```

Comment:
The usage of underscore in this context is not too obvious, even for me.
Maybe add a comment why we have this convention in the first place?

Reply:
Upon checking the code, this is how we generate the `uniqueKey` (file name of image / video). The `key` is the actual file name of the uploaded image (`e.g. geli.png`). It will be appended by a `random value` separated by a comma. As a result, the file name uploaded to the s3 and saved to the database follows this format. (e.g. `0xlofisl_geli.png`);
<img width="332" alt="Screenshot 2023-08-19 at 11 42 07 AM" src="https://github.com/klaudsol/klaudsol-cms/assets/101687552/932040a2-f012-46f0-9713-eba626328a53">

Then in the `EntityAttributeValue.js` file (This file formats the values fetched from the DB), there exists a function named `formatImage` this is similar to the function that I have created. 
<img width="584" alt="Screenshot 2023-08-19 at 11 40 40 AM" src="https://github.com/klaudsol/klaudsol-cms/assets/101687552/08ebf2ae-c410-449a-bdc1-26536b1e76f4">

Reply:
Perfect, can you add these into the codebase as a comment? :)